### ISSUE TITLE
Unclear which of "rst" or "riseset" functions are recommended

### ISSUE THREAD

**Issue by tbrowder** [skrushinsky/astro-montenbruck#18]

**Comment by skrushinsky** [comment]
Well, they come from the same source, i.e. _"Astronomy on the PC"_ by O.Montenbruck and T.Pfleger, but are built on different algorithms: `riseset` utilizes quadratic interpolation while `rst` is iterative.  Along with rise and set, `rst`  gives transit times. At the other hand, `riseset` is a base for calculating twilight. 

I have not compared yet, which one is more precise or performant.  

Do you think, it makes sense to add this to the documentation?

**Comment by tbrowder** [comment]
Yes, that would be helpful at least to me. And I see both scripts show transit time.
